integration_test/testdata/nginx-example/templates/NOTES.txt

```diff
@@ -20,3 +20,5 @@
   echo "Visit http://127.0.0.1:8080 to use your application"
   kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:$CONTAINER_PORT
 {{- end }}
+
+BTW, lookup saw {{ (lookup "test.stackrox.io" "FakeResource" .Release.Namespace "").items | len }} FakeResource
```

Comment:
Is this debug code?

Reply:
Not really. As I understand this is a toy chart that acts as an integration test for helmtest. So I added functionality that exercises the new feature to prevent regressions.

Reply:
Comments on how to do it better most welcome :-)

Reply:
LGTM :D